deeplake/core/dataset/dataset.py

```diff
@@ -3826,6 +3826,10 @@ def connect(
         self._load_link_creds()
         self._first_load_init(verbose=False)
 
+        next_storage = self.storage.next_storage
```

Comment:
We can use `from deeplake.util.remove_cache import get_base_storage` to get the base storage